blockindex/indexpb/index.proto

```diff
@@ -17,3 +17,9 @@ message BlockIndex {
 message ActionIndex {
     uint64 blkHeight = 1;
 }
+
+message SGDIndex {
+    string contract = 1;
+    string receiver = 2;
```

Comment:
add `deployTime`

Reply:
There is no way to get `deployTime`, all contracts `deploytime` needs to be recorded with indexer(same as before). 
Maybe it can be filled when approving in web form and passed in through the contract